e2e/tests/text/adding-text/index.spec.ts

```diff
@@ -6,7 +6,9 @@ import { TextUtil } from '../../../utils/TextUtil';
 
 setup();
 
-[false, true].forEach((splitByGrapheme) => {
+test.describe.configure({ mode: 'serial' });
```

Comment:
this is to avoid that we run 2 fabricJS tests in the same browser window and we mix up the copy paste status that is stored globally.
Or at least it solved that for me since code that was behaving normally when using my hands wasn't behaving the same when using the tests in parallel.
Should affect ONLY this file here.